core/drivers/crypto/hisilicon/include/hisi_qm.h

```diff
@@ -0,0 +1,274 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2022, Huawei Technologies Co., Ltd
+ */
+#ifndef __HISI_QM_H__
+#define __HISI_QM_H__
+
+#include <initcall.h>
```

Comment:
initcall.h is not needed by this header file.
It is used only from some C source file. Please include the header only from the source file that needs it.

Ditto for other included header : include them only from the H or C files that need them.